reliability-engineering/terraform/modules/concourse-worker-pool/files/worker-init.sh

```diff
@@ -116,3 +116,36 @@ iptables -P FORWARD ACCEPT
 
 apt-get install --yes prometheus-node-exporter
 systemctl enable --now prometheus-node-exporter
+
+## install cloudwatch log agent
+curl -o /root/amazon-cloudwatch-agent.deb https://s3.amazonaws.com/amazoncloudwatch-agent/debian/amd64/latest/amazon-cloudwatch-agent.deb
```

Comment:
nit: do we want to pin?

Reply:
Wanted to keep this consistent with the agent in `concourse-web` which seems to pull this.... maybe we should pin both in the future